tests/test_ops/test_rotated_feature_align.py

```diff
@@ -128,3 +128,122 @@ def test_rotated_feature_align():
     output.backward(torch.ones_like(output))
     assert torch.allclose(output, expected_output, 1e-2)
     assert torch.allclose(feature.grad, expected_grad, 1e-2)
+
+
+def test_rotated_feature_align_cpu():
```

Comment:
We do not need to create another test function. A new parameter is enough. `test_roi_align` as example.
```python
@pytest.mark.parametrize('device', ['cuda', 'cpu'])
@pytest.mark.parametrize('dtype', [torch.float, torch.double, torch.half])
def test_roialign(device, dtype):
```

Reply:
Done